src/app.ts

```diff
@@ -10,6 +10,7 @@ import cors from 'fastify-cors'
 function HospitalRun(fastify: FastifyInstance, opts: any, next: nextCallback) {
   fastify.register(cors, {
     allowedHeaders: ['Content-Type', 'Authorization'],
+    preflight: false,
```

Comment:
We actually need this to be true. Why changing it to false?

Reply:
@jackcmeyer